src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/SpecialVideosPageObject.java

```diff
@@ -65,24 +66,12 @@ public WatchPageObject unfollowVideo(String wikiURL, String videoName) {
     return new WatchPageObject(driver);
   }
 
-  private void verifyH1() {
-    wait.forElementVisible(h1Header);
-  }
-
-  private void verifyNewestVideo() {
-    wait.forElementVisible(newestVideo);
-  }
-
-  private void verifyAddVideoButton() {
-    wait.forElementClickable(addVideo);
-  }
-
   private void verifySortDropdown() {
     wait.forElementVisible(sortDropdown);
   }
 
   public VetAddVideoComponentObject clickAddAVideo() {
-    verifyAddVideoButton();
+    wait.forElementClickable(addVideo);;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)